packages/manager/modules/common-translations/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@ovh-ux/manager-common-translations",
+  "version": "0.1.0",
+  "private": true,
+  "description": "Common translations for Manager µ-apps",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ovh/manager.git",
+    "directory": "packages/manager/modules/common-translations"
+  },
+  "license": "BSD-3-Clause",
+  "author": "OVH SAS",
+  "sideEffects": false,
+  "main": "./dist/index.cjs.js",
+  "module": "./dist/index.esm.js",
+  "types": "./dist/types/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "tsc && vite build",
```

Comment:
Question: As we are only handling here `json` files, I am wondering if it's mandatory to have tsc in the scope?

I mean, there is no type checking and no declaration files generated as we just copy static elements into the dist folder.

By the way, if needed, `typescript` should be in scope to run tsc scripts.

Reply:
There's a TS file which exports the namespaces https://github.com/ovh/manager/pull/14752/files#diff-1185723c5f7c95552f7c4b7f4020358e42d516c528f68949c24147533f15ce0c. This way it will be easy to consume with i18n.

Reply:
Yes, but is it mandatory to use tsc and vite-dts (that have a cost on build phase) to produce a simple raw object?

Reply:
It is to assist the developers with the available namespaces from the package with the auto-completion.